nvflare/app_common/widgets/log_streaming.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
I thought we talked about splitting this into 3 separate files:
- A common file that contains the definition of channel and other constants, and potentially a base class for all senders.
- A file that implements error log sender;
- A file that implements the receiver.

Please follow this structure.